File: Ruby/linearAlgorithm.rb
def linear_search(list, number)
    list.each_with_index do |index, elt|
        if elt == number
            index
        end
      nil
    end
end

def verrify_number(number)
    return "Not found" unless number
      return "number exist on the index #{number}"
    end
end

numbers = [1,2,3,4,5,6,7,8,9,10]
numb = linear_search(numbers, 3)
puts verrify_number(3)

File: Js/linearAlgorithm.js
const linearSearch = (list, index) => {
    for(let i=0; i<=list.length; i++) {
        if (index === list[i]){
            return i
        }
    }
    return null
}

const verifyValue = (index) => {
    if(index) {
        return `Value exists on index ${index}`
    }
   return "Not found"

}

const numbers = [1,2,3,4,5,6,7,8,9,10]
const numb = linearSearch(numbers, 42)
console.log(verifyValue(numb))